tsconfig.base.json

```diff
@@ -0,0 +1,54 @@
+{
+	"compilerOptions": {
+		"outDir": "./build",
+		"sourceMap": true,
+		"baseUrl": ".",
+		"module": "esnext",
+		// Don't emit; allow Babel to transform files
+		"noEmit": true,
```

Comment:
I think this should `emitDeclarationOnly` to better support a composite repo. I believe that's how referencing projects access type information.